app/controllers/users_controller.rb

```diff
@@ -4,12 +4,16 @@ class UsersController < ApplicationController
 
   def add_pregnancy
     @user.pregnancies << @pregnancy
-    respond_to { |format| format.js }
+    respond_to do |format|
```

Comment:
Great call here -- I think this sidesteps a lot of JQuery problems we'd have otherwise.